packages/builder/src/components/automation/SetupPanel/RowSelectorTypes.svelte

```diff
@@ -14,6 +20,8 @@
   export let value
   export let bindings
   export let isTestModal
+  export let meta
```

Comment:
It looks like you are only using `meta` for `meta?.fields?.[field]?.`. This long null checks get things a little bit hard to follow. We could either pass the actual field metadata as input (instead of the full meta) or extract the value as `$: useAttachmentBinding = meta?.fields?.[field]?.useAttachmentBinding` and use that instead